### PR TITLE
Style ProjectPreview Cards

### LINKED ISSUE #13: Implement project preview list on the front page
This page: https://xd.adobe.com/view/395518bf-7e51-4eb2-6ff2-3f1b57fdaa17-4f64/

### PR DESCRIPTION
**Pull request by positiveimpact**
Restyled the ProjectPreview cards. Made them more friendly by putting the info under the image. That makes it much easier to display them next to each other on different screen sizes. 
Images like sneep.png and canteen.png will be put into public/images once #49 gets merged.
Closes #13

### CODE REVIEW

File: projects.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
     {
       "id": "78b6677c-28d2-4b2b-9825-4479706d2402",
       "name": "CO2-Labels for University Canteen",
-      "image": "/data_temp/images/canteen.png",
+      "image": "canteen.png",
```
Comment (EvanHahn) [inline projects.json]: Should these use absolute URLs (like `/canteen.png`)?

File: src/components/layout.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ export default function Layout({ title, children }) {
         <Header />
         <Container maxWidth="lg" component="main">
           <Container maxWidth="sm">
-            <Typography variant="h1" className={classes.mainHeading}>
+            <Typography variant="h5" className={classes.mainHeading}>
```
Comment (EvanHahn) [inline src/components/layout.js]: This doesn't seem related to this PR. Why did this change?

File: src/components/project/ProjectMetaData.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+});
+
+export default function ProjectMetaData({ project }) {
+  const classes = useStyles();
+
+  return (
+    <Box>
+      <Box>
+        <span className={classes.cardIconBox}>
+          <img src={project.organisation_image} className={`${classes.orgLogo} `} />
```
Comment (EvanHahn) [inline src/components/project/ProjectMetaData.js]: Nitpick: I think this can just be `className={classes.orgLogo}`.

File: src/components/project/ProjectMetaData.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    display: "inline-block"
+  },
+  cardIcon: {
+    verticalAlign: "bottom",
+    marginBottom: -2,
+    marginTop: 2
+  }
+});
+
+export default function ProjectMetaData({ project }) {
```
Comment (EvanHahn) [inline src/components/project/ProjectMetaData.js]: I'm fine to have this in a separate component, but it seems like it's really only useful in the `ProjectPreview` component. Why a separate component/file?

File: src/components/project/ProjectPreview.js [hunk 5]
```diff
@@ -1,29 +1,31 @@
...
+  return {
+    root: {
+      "&:hover": {
+        cursor: "pointer"
+      },
+      "-webkit-user-select": "none",
+      "-moz-user-select": "none",
+      "-ms-user-select": "none",
+      userSelect: "none",
+      backgroundColor: "#fafafa",
```
Comment (EvanHahn) [inline src/components/project/ProjectPreview.js]: Should this color come from a theme?

File: src/components/layout.js (hunk 2 above)
Comment (positiveimpact) [reply src/components/layout.js]: I changed it because I thought it looked oversized compared to the new card/font sizes. I added component="h1" to it. Would you like me to move it to a different PR?

File: src/components/project/ProjectMetaData.js (hunk 4 above)
Comment (positiveimpact) [reply src/components/project/ProjectMetaData.js]: I thought it makes it a bit clearer. If you think it makes more sense to not create new files for components, that are only used once, I will move it back to the main file.

File: src/components/project/ProjectPreview.js (hunk 5 above)
Comment (positiveimpact) [reply src/components/project/ProjectPreview.js]: That was the color where I had trouble accessing it and you suggested it might be in CssBaseline. I changed it to backgroundColor: "inherit" which resolves to the same color.

**Comment by positiveimpact** [comment]
Thanks for the reviews! I made changes and left comments about changes I didn't make (yet). 
Let me know if you think further changes should be made.
